bin/default_pythons

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+DEFAULT_PYTHON_VERSION="python-3.6.8"
+LATEST_36="python-3.6.8"
+LATEST_37="python-3.7.2"
+LATEST_35="python-3.5.6"
+LATEST_34="python-3.4.9"
+LATEST_27="python-2.7.16"
+
+export DEFAULT_PYTHON_VERSION LATEST_37 LATEST_36 LATEST_35 LATEST_34 LATEST_27
```

Comment:
I don't think you actually need to export the env vars at all unless they are going to be read by a separate process. It's unlikely to hurt though.